cli/installer/install-azd.sh

```diff
@@ -113,8 +113,12 @@ get_architecture() {
     if [ "$architecture_raw" = "x86_64" ]; then
         echo 'amd64';
         return;
+    elif [ "$architecture_raw" = "aarch64" ] && [ "$platform" = 'linux' ]; then
+        # Translate "aarch64" to "arm64" when building Linux URL
+        echo 'arm64';
+        return;
     elif [ "$architecture_raw" = "arm64" ] && [ "$platform" = 'darwin' ]; then
-        # In the case of Apple Silicon use the existing ARM64 environment
+        # In the case of Apple Silicon use the existing ARM64 environment with Rosetta 2
```

Comment:
```suggestion
        # In the case of Apple Silicon use the existing AMD64 environment with Rosetta 2
```

Reply:
I think this more accurately describes what we are trying to do here (use the x64 binary under Rosetta, instead of a native arm64 version)